tests/realtikvtest/loaddatatest/util_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package loaddatatest
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/fsouza/fake-gcs-server/fakestorage"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/testkit"
+	"github.com/pingcap/tidb/tests/realtikvtest"
+	"github.com/stretchr/testify/suite"
+)
+
+type mockGCSSuite struct {
+	suite.Suite
+
+	server *fakestorage.Server
+	store  kv.Storage
+	tk     *testkit.TestKit
+}
+
+var (
+	gcsHost = "127.0.0.1"
+	gcsPort = uint16(4443)
```

Comment:
need change to another port to reduce conflict, maybe we can use [freeport](https://github.com/phayes/freeport) in future.

And seems this test suite is very common, we can provide a constructor for it in future

Reply:
By the way, when running the run-real-tikv, we only run with a simple goroutine in the container. It is not a severe problem.